conf/broker.conf

```diff
@@ -29,13 +29,13 @@ globalZookeeperServers=
 brokerServicePort=6650
 
 # Broker data port for TLS
-brokerServicePortTls=6651
+brokerServicePortTls=
```

Comment:
Since by default TLS is diabled

Reply:
Just to confirm, you mean TLS is disabled by a preset value(null) and non-TLS port will be disabled if I set `brokerServicePort` to null, right?